lib/rules/no-global-jquery.js

```diff
@@ -3,6 +3,10 @@
  */
 'use strict';
 
+const { getEmberImportBinding } = require('../utils/imports');
+const { collectObjectPatternBindings } = require('../utils/destructed-binding');
+
+const MESSAGE = 'Do not use global `$` or `jQuery`. Use `Ember.$` instead. Please see the following guide for more information: https://github.com/ember-best-practices/eslint-plugin-ember-best-practices/blob/master/guides/rules/no-global-jquery.md';
```

Comment:
I know that this is not new, but it seems odd that we are advocating for folks to use `Ember.$` here but have other rules to roughly disallow it also...